packages/test-utils/src/wrapper.js

```diff
@@ -112,9 +115,17 @@ export default class Wrapper implements BaseWrapper {
         }
       })
       classes = classes.map(
-        className => cssModuleIdentifiers[className] || className
+        klass => cssModuleIdentifiers[klass] || klass
```

Comment:
Why the name change? Can you replace it with `c` rather than `klass`

Reply:
`className` ended up shadowing the argument name so I ended up changing it here since it isn't user facing. I am not sure you can use `class` since it is a reserved word. How about `name`?